ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -160,7 +159,6 @@ tests:
         NUM_MASTERS=1
         NUM_WORKERS=0
         ISO_IMAGE_TYPE=minimal-iso
-        OPENSHIFT_INSTALL_RELEASE_IMAGE=${RELEASE_IMAGE_LATEST}
```

Comment:
this is periodic

Reply:
That's fine. It would be set by assisted-setup
https://github.com/openshift/release/blob/ab453f2d7bffbc21de6afa0151ae5b7241bb48de/ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh#L87-L88